dev/envvars.sh

```diff
@@ -53,9 +53,12 @@ then
     export CORECLR_PROFILER_PATH_64="${CURDIR}/bin/tracer-home/win-x64/OpenTelemetry.AutoInstrumentation.Native.${SUFIX}"
     export CORECLR_PROFILER_PATH_32="${CURDIR}/bin/tracer-home/win-x86/OpenTelemetry.AutoInstrumentation.Native.${SUFIX}"
 fi
+
+# Configure .NET Core runtime
+export DOTNET_ADDITIONAL_DEPS="${CURDIR}/bin/tracer-home/AdditionalDeps"
```

Comment:
`DOTNET_ADDITIONAL_DEPS` should be supported by additional environmental variable `DOTNET_SHARED_STORE`. Only adding additional deps will not make any difference. We need to update this location and in md files.

Ref: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/USAGE.md#additional-deps-and-runtime-package-store

Reply:
Yeah. I have forgotten about it. Nice catch. 👍

Reply:
@rajkumar-rangaraj Is the recommendation to only set those environment variables when necessary? Or is this something that we should set by default?

Reply:
@nrcventura  For now, I would recommend setting it even when it is not necessary as it should not cause any harm.

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/374/commits/a90c48d0e69e725bb04b6851f348f11512c074cd